### ISSUE TITLE
Does this library support Classic Auction Data?

### ISSUE THREAD

**Issue by cosileone** [benweier/blizzard.js#166]
Looks like blizzard added support for Auction House data in the [classic game data APIs](https://develop.battle.net/documentation/world-of-warcraft-classic/game-data-apis) at some point. 

Does this library support those endpoints yet?

**Comment by benweier** [comment]
Thanks for letting me know @cosileone 😄 I'll look into these docs and update if necessary

**Comment by benweier** [comment]
Hi @cosileone I have published `4.4.0` which includes the Classic WoW Auction House method.

```
classic.auctionHouse({ realm: 123 })
classic.auctionHouse({ realm: 123, auctionHouse: 2 })
```

**Comment by cosileone** [comment]
Awesome, thank you very much for being active on this project! It is very much appreciated
